app/core/base.js

```diff
@@ -157,6 +157,10 @@ FauxtonAPI.setSession = function (newSession) {
   return FauxtonAPI.session.fetchUser();
 };
 
-FauxtonAPI.reducers = [];
+FauxtonAPI.reducers = {};
+
+FauxtonAPI.addReducers = (reducers) => {
+  FauxtonAPI.reducers = Object.assign({}, FauxtonAPI.reducers, reducers);
+};
 
```

Comment:
This not the most elegant solution but reducers are named space by an object so when we add reducer to the store we merge it with previous reducers object and reassign it